servers/src/main/java/tachyon/master/rawtable/RawTableMasterServiceHandler.java

```diff
@@ -46,6 +55,21 @@ public long createRawTable(String path, int columns, ByteBuffer metadata)
   }
 
   @Override
+  public long createTable(final String path, final int columns, final ByteBuffer metadata, String nonce)
+      throws TachyonTException, ThriftIOException, TException {
+    try {
+      return ((Long) replayCache.get(nonce, new Callable<Object>() {
```

Comment:
Definitely - for now I'm just doing it in this one place to fix the flaky build. When we extend the pattern to all non-idempotent operations we can have more discussion about the best way to do it.